web-components/src/components/icons/RegenMarketIcon.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import SvgIcon from '@mui/material/SvgIcon';
+
+interface Props {
+  color?: string;
+  className?: string;
```

Comment:
I think this SVG was created based on an old one.
New one use `sx` instead of className.
We probably don't need to use a prop color also use 'currentColor' instead.
We don't need to use the SVG tag twice, it will be added by the `<SVGIcon />`.
Finally, it's best not to set hardcoded width/height directly, just setting the viewport is enough.

For a more up-to-date example you can check this one for example https://github.com/regen-network/regen-web/blob/master/web-components/src/components/icons/EmptyCartIcon.tsx